lib/ansible/modules/inventory/add_host.py

```diff
@@ -49,6 +49,13 @@
     groups: just_created
     foo: 42
 
+# add a host to multiple groups
```

Comment:
use 'name' task keyword as a better way to 'comment on tasks'

Reply:
Done. Thanks for reviewing.